R/misc.R

```diff
@@ -0,0 +1,16 @@
+
+organize_glmnet_pred <- function(x, object) {
+  if (ncol(x) == 1) {
+    res <- x[, 1]
+    res <- unname(res)
+  } else {
+    n <- nrow(x)
+    res <- utils::stack(as.data.frame(x))
+    if (!is.null(object$spec$args$penalty))
+      res$lambda <- rep(object$spec$args$penalty, each = n) else
+        res$lambda <- rep(object$fit$lambda, each = n)
+    res <- res[, colnames(res) %in% c("values", "lambda")]
+  }
+  res
+}
```

Comment:
this function is exported from parsnip as `.organize_glmnet_pred()` 

https://github.com/tidymodels/parsnip/blob/37b07cbe609b6799c4d2a2b2bc3c5430f6b1f835/R/linear_reg.R#L176-L189

Reply:
and a copy is also in `R/poisson_reg.R`

Reply:
I don't think that the `translate()` or `check_args()` functions should move. If one of the parsnip-adjacent packages add a new engine, we shouldn't have to modify parsnip.